package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@json-editor/json-editor",
+  "name": "@myporsche/pid-json-editor",
```

Comment:
@florina-chesaru Umm, I think this one went on the wrong remote.

Reply:
@florina-chesaru I think you can `push --force` to remove it or create a new PR.

Reply:
Ooops, sorry for that :-)

Reply:
No problem ;)